subprojects/core/src/integTest/groovy/org/gradle/operations/configuration/ProjectIsolationSettingsBuildOperationsIntegrationTest.groovy

```diff
@@ -0,0 +1,71 @@
+/*
+ * Copyright 2023 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.gradle.operations.configuration
+
+import org.gradle.integtests.fixtures.AbstractIntegrationSpec
+import org.gradle.integtests.fixtures.BuildOperationsFixture
+import org.gradle.integtests.fixtures.executer.GradleContextualExecuter
+import org.gradle.internal.configurationcache.options.ConfigurationCacheSettingsFinalizedProgressDetails
+
+class ProjectIsolationSettingsBuildOperationsIntegrationTest extends AbstractIntegrationSpec {
```

Comment:
:x: It should be called `IsolatedProjects*` for consistency with the existing code.